source/leuven/wice_advanced_guide.rst

```diff
@@ -69,12 +38,11 @@ use a specific module:
 
 .. code-block:: shell
 
-    $ module use /apps/leuven/skylake/2021a/modules/all
     $ module load worker/1.6.12-foss-2021a-wice
 
 If instead you want to launch Worker jobs from an interactive job running on
 wICE, you can use the ``worker/1.6.12-foss-2021a`` module (but do make sure
-this is the version installed in a subdirectory of ``/apps/leuven/icelake``).
+this is the version installed *specifically* for wICE.
```

Comment:
How can users make sure?

Reply:
Done in https://github.com/hpcleuven/VscDocumentation/commit/23c33e84b6452c00c70a7c41cc1e70a8980fb162